### PR TITLE
Ash/master

### PR DESCRIPTION
**Pull request by annaleeherrera**
Sinatra Site Wave 1

### CODE REVIEW

File: my-site.rb [hunk 1]
```diff
@@ -2,7 +2,15 @@
 
 class MySite < Sinatra::Base
 
-  get "/" do
```
Comment (kariabancroft) [inline my-site.rb:5]: It's still good to have the default "root route" in your page so that a user will be redirected to _something_ if they don't navigate to an appropriate URL

File: views/food.erb [hunk 2]
```diff
@@ -0,0 +1,78 @@
+<html>
```
Comment (kariabancroft) [inline views/food.erb:1]: Watch out for indentation when you use nested tags, once you start nesting, each subsequent tag should be indented one level. so here with the body and then on lines 4,5,7,etc.

File: views/food.erb [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+<footer>
+
+  <p>
+    Get your Seattle on:
+    <a href = "/loveseattle.html">
+    Home</a>
+    <a href ="/seattleparks.html">
+    Parks</a>
+
```
Comment (kariabancroft) [inline views/food.erb:73]: Also watch out for extra unnecessary whitespace - I'd remove any lines where you have two lines of whitespace

File: views/hello.erb [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<html>
```
Comment (kariabancroft) [inline views/hello.erb:1]: Same comment re: indentation here

File: my-site.rb (hunk 1 above)
Comment (annaleeherrera) [inline my-site.rb:5]: Root route, as in the home page?

File: views/food.erb (hunk 3 above)
Comment (annaleeherrera) [inline views/food.erb:73]: Will go through whole document removing any whitespace.

File: views/hello.erb (hunk 4 above)
Comment (annaleeherrera) [inline views/hello.erb:1]: Will go through whole document fixing indentations.

File: views/food.erb (hunk 2 above)
Comment (annaleeherrera) [inline views/food.erb:1]: Will scan the rest of the document.  Thanks !

File: my-site.rb (hunk 1 above)
Comment (kariabancroft) [inline my-site.rb:5]: Yes, just the regular localhost:xxxx/
